doc/user/options.md

```diff
@@ -112,14 +113,15 @@ variables when running on a JVM (except for `JAVACMD` on the GraalVM).
 
 ## SVM-specific switches
 
-The SVM supports `-D` for setting system properties and `-XX:arg` for SVM
-options. Unlike with the standard Ruby command-line, these options are always
+The SVM supports `--native.D` for setting system properties and 
+`--native.XX:arg` for SVM options. 
+Unlike with the standard Ruby command-line, these options are always
 taken by the SVM, wherever they appear in the arguments (such as after a `--`).
```

Comment:
Is this still true?
I think it applied to the old SVM Main without launcher which has a special parsing of `-D`.

Reply:
Indeed, that's outdated